be/src/formats/parquet/column_chunk_reader.cpp

```diff
@@ -105,6 +105,9 @@ Status ColumnChunkReader::_parse_page_header() {
     if (_page_reader->current_header()->type == tparquet::PageType::DATA_PAGE) {
         const auto& header = *_page_reader->current_header();
         _num_values = header.data_page_header.num_values;
+        _opts.stats->has_page_statistics |=
+                (header.data_page_header.__isset.statistics && (header.data_page_header.statistics.__isset.min_value ||
+                                                                header.data_page_header.statistics.__isset.min));
```

Comment:
do we need to check if max exist or not

Reply:
I think it's ok, usually set min/max together, just check min or max is enough.